alibi_detect/saving/loading.py

```diff
@@ -139,7 +141,16 @@ def _load_detector_config(filepath: Union[str, os.PathLike]) -> ConfigurableDete
     # Init detector from config
     logger.info('Instantiating detector.')
     detector = _init_detector(cfg)
+
+    # Load state if it exists (and detector supports it)
+    # TODO - this will be removed in follow-up offline state PR, as loading to be moved to __init__ (w/ state_dir kwarg)
+    if isinstance(detector, StatefulDetectorOnline):
+        state_dir = config_dir.joinpath((STATE_PATH))
```

Comment:
Nitpick: my IDE says the extra parentheses are unnecessary.

Reply:
Corrected in da5671403b2ec500a577935db4e25618746974a7.